espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/exceptions/StaleElementException.java

```diff
@@ -0,0 +1,7 @@
+package io.appium.espressoserver.lib.handlers.exceptions;
+
+public class StaleElementException extends AppiumException {
+    public StaleElementException(String elementId) {
+        super(String.format("Element %s no longer exists", elementId));
```

Comment:
I observe that people usually don't understand the meaning of this error and have no idea how to fix it. What if we put more details into exception message, for example:

``` 
The cached element %s no longer exists in the accessibility objects hierarchy. Try to find it using its locator.
```